analyzers/src/SonarAnalyzer.CSharp/Rules/PasswordsShouldBeStoredCorrectly.cs

```diff
@@ -98,6 +109,15 @@ private static void Rfc2898DeriveBytes(SonarAnalysisContext context)
             "Use at least 100,000 iterations and a state-of-the-art digest algorithm here.",
             objectCreationTracker.MatchConstructor(KnownType.System_Security_Cryptography_Rfc2898DeriveBytes),
             x => x.InvokedConstructorSymbol.Value.Parameters.All(x => x.Name != "hashAlgorithm"));
+
+        var propertyTracker = CSharpFacade.Instance.Tracker.PropertyAccess;
+        Track(
+            propertyTracker,
+            context,
+            UseMoreIterationsMessageFormat,
+            propertyTracker.MatchSetter(),
+            propertyTracker.MatchProperty(new MemberDescriptor(KnownType.System_Security_Cryptography_Rfc2898DeriveBytes, "IterationCount")),
+            x => HasFewIterations(x, propertyTracker));
```

Comment:
Remainder from last PR, tracking the IterationCount property access on an RfcDerivedBytes object